qa/evil-tests/build.gradle

```diff
@@ -27,3 +27,12 @@ dependencies {
 tasks.named("test").configure {
   systemProperty 'tests.security.manager', 'false'
 }
+
+if (buildParams.inFipsJvm) {
+  // Disable evil tests in FIPS mode due to jar hell between plugin-cli's
+  // BouncyCastle dependencies and the BC FIPS dependencies added by the FIPS gradle config
+  // TODO: re-evaluate this after updating plugin-cli to use BC FIPS 2.x
+  tasks.named("test").configure {
```

Comment:
Looks like the plugin-cli PR is close to being merged https://github.com/elastic/elasticsearch/pull/138949 should this be removed or at least have a jira so it's not forgotten?